nix/default.nix

```diff
@@ -2,32 +2,32 @@
 , crossSystem ? null
 , config ? {}
 , sourcesOverride ? {}
+, gitrev ? null
 }:
 let
   sources = import ./sources.nix { inherit pkgs; }
     // sourcesOverride;
-  iohKNix = import sources.iohk-nix {};
-  haskellNix = import sources."haskell.nix";
+  iohkNix = import sources.iohk-nix {};
```

Comment:
I did the same change `iohKNix` → `iohkNix` on cardano-wallet and this change broke our macos release.
There is already an `iohkNix` variable below.
This is nasty. It took a git bisect to find it.
I suggest renaming `iohKNix` → `iohkNixMain` to avoid potential problems.

Reply:
😮  Thanks 😁